samples/subsys/fs/fs_sample/boards/mg100.conf

```diff
@@ -1,2 +1,2 @@
 CONFIG_SPI=y
-CONFIG_DISK_DRIVER_SDMMC=y
+CONFIG_REGULATOR=y
```

Comment:
Would it make sense to default `CONFIG_REGULATOR` to `y` on board level based on `CONFIG_DISK_DRIVER_SSDMMC` value?

Reply:
i do like that idea...
Fix incoming

Reply:
Works!

```
*** Booting Zephyr OS build zephyr-v3.4.0-2311-gda50f1be3c75 ***
[00:00:00.363,372] <inf> sd: Maximum SD clock is under 25MHz, using clock of 8000000Hz
[00:00:00.364,318] <inf> main: Block count 7806976
Sector size 512
Memory Size(MB) 3812
Disk mounted.

Listing dir /SD: ...
[DIR ] SPOTLI~1
[DIR ] FSEVEN~1
```